comps/guardrails/pii_detection/test.py

```diff
@@ -28,7 +28,7 @@ def test_html(ip_addr="localhost", batch_size=20):
             print("An error occurred:", e)
 
 
-def test_text(ip_addr="localhost", batch_size=20):
+def test_text(ip_addr="localhost", batch_size=20, strategy=None):
     proxies = {"http": ""}
     url = f"http://{ip_addr}:6357/v1/piidetect"
     if os.path.exists("data/ai_rss.csv"):
```

Comment:
should you upload an example file for ai_rss.csv? Otherwise this part of code is no meaning here.

Reply:
I deleted these lines and changed to another url opea.dev for testing

Reply:
it seems still a check for "data/ai_rss.csv". What is purpose of these code if there is no such a file.